app/models/html_storage.rb

```diff
@@ -0,0 +1,16 @@
+class HtmlStorage < DocumentStorage
```

Comment:
Let's move this file from `app/models/html_storage.rb` to `app/models/document_storage_type/html.rb`.

You'll have to change this line to `class DocumentStorageType::Html < DocumentStorage`, or if that doesn't work try:
```
module DocumentStorageType
  class Html < DocumentStorage
```
I'd add a unit test to double check.